runtime/sam/expr/function/time.go

```diff
@@ -46,3 +49,49 @@ func (b *Bucket) Call(ectx expr.Context, args []zed.Value) zed.Value {
 	}
 	return zed.NewTime(nano.Ts(v).Trunc(bin))
 }
+
+var strftimeSet strftime.SpecificationSet
+
+func init() {
+	strftimeSet = strftime.NewSpecificationSet()
+	strftimeSet.Delete('n') // newline
+	strftimeSet.Delete('R') // equivalent to %H:%M
+	strftimeSet.Delete('r') // equivalent to %I:%M:%S %p
+	strftimeSet.Delete('t') // tab
+	strftimeSet.Delete('T') // equivalent to %H:%M:%S
+	strftimeSet.Delete('v') // equivalent to %e-%b-%Y
+	strftimeSet.Delete('V') // the week number of the year (Monday as the first day of the week) as a decimal number (01-53)
+	strftimeSet.Delete('z') // timezone
+	strftimeSet.Delete('Z') // timezone
+}
+
+// https://github.com/brimdata/zed/blob/main/docs/language/functions.md#strftime
+type Strftime struct {
+	zctx      *zed.Context
+	format    string
+	formatter *strftime.Strftime
+}
+
+func (s *Strftime) Call(ectx expr.Context, args []zed.Value) zed.Value {
+	formatArg := args[0]
+	if !formatArg.IsString() {
+		return s.zctx.WrapError(ectx.Arena(), "strftime: string value required for format arg", formatArg)
+	}
+	if formatArg.IsNull() {
+		return zed.NullString
+	}
+	format := formatArg.AsString()
+	v, ok := coerce.ToInt(args[1])
+	if !ok {
+		return s.zctx.WrapError(ectx.Arena(), "strftime: time value required for time arg", args[1])
+	}
```

Comment:
Using coerce.ToInt here results in behavior that's inconsistent with both this error message and the documentation in docs/language/functions/strftime.md.